docs/UserGuide.md

```diff
@@ -206,25 +206,36 @@ _Details coming soon ..._
+| **List all events**                           | `elist`                                                                                                                                                                                                                                                                                |
+| **Read an individual event**                  | `eshow EVENT_ID` <br> e.g., `eshow 8`                                                                                                                                                                                                                                                  |
+| **Delete an event**                           | `edelete EVENT_ID` <br> e.g., `edelete 3`                                                                                                                                                                                                                                              |
+| **Create a new volunteer profile**            | `vcreate vn/VOLUNTEER_NAME hp/PHONE_NUMBER e/EMAIL [s/SKILLS]...`<br> e.g.,`vcreate vn/John Lim hp/81234567 e/john123@gmail.com s/Cooking`                                                                                                                                             |
+| **List all volunteer profiles**               | `vlist`                                                                                                                                                                                                                                                                                |
+| **Delete a volunteer profile**                | `vdelete VOLUNTEER_ID` <br> e.g., `vdelete 4`                                                                                                                                                                                                                                          |
+| **Add a volunteer to an event**               | `eaddv vid/VOLUNTEER_ID /to eid/EVENT_ID`<br> e.g., `eaddv vid/1 /to eid/3` <br> Alternatively, `eaddv vn/VOLUNTEER_NAME /to en/EVENT_NAME` <br> e.g., `eaddv vn/Betsy Crowe /to en/Fundraising for needy families`                                                                    |
+| **Check for volunteers assigned to an event** | `elistv eid/EVENT_ID` <br> e.g. `elistv eid/8` <br> Alternatively, `elistv en/EVENT_NAME` <br> e.g., `elistv en/October beach clean up`                                                                                                                                                |
+| **Remove a volunteer from an event**          | `eremovev vid/VOLUNTEER_ID /from eid/EVENT_ID`<br> e.g., `eremovev vid/3 /from eid/4` <br> Alternatively, `eremovev vn/VOLUNTEER_NAME /from en/EVENT_NAME` <br> e.g., `eremovev vn/John Lim /from en/October beach clean up`                                                           |
```

Comment:
Based on what we discussed earlier, i think the /from can be removed.

Reply:
okai will make the necessary changes